cfgov/legacy/static/knowledgebase/less/es-ask-base.less

```diff
@@ -1,3 +1,137 @@
+/* Changes to make ES look like English Ask */
+
+body {
+    font-family: "Avenir Next", Arial, sans-serif;
+}
+
+.wrapper-body .wrapper-container {
+    padding-top: 0px;
+}
+
+.ac-search {
+    padding: 30px;
+    .lead {
+        text-transform: none;
+        font-weight: 400;
+        font-size: 22px;
+        font-family: "Avenir Next", Arial, sans-serif;
+        line-height: 1.2727;
+        margin: 0 0 0.68em;
+        max-width: 670px;
+    }
+}
+
+.full-bleed.content {
+    margin-top: 0em;
+}
+
+.es-ask__results-section {
```

Comment:
Is this correct BEM? (Block-Element__Modifier) So `__results-section` is a modifier on the `es-ask` block. Should this actually be `es-ask_results-section`?

Reply:
Ooos, good catch, @anselmbradford